uint8 status
---
bool confirmation